app/views/solutions/show.html.haml

```diff
@@ -63,7 +63,7 @@
               .legacy-warning
                 %h4 Note:
                 -if @solution.published_at <= Exercism::V2_MIGRATED_AT
-                  This solution was written on an old version of Exercism. The tests show below might not corrospond to the solution code, and the exercise may have changed since this code was written.
+                  This solution was written on an old version of Exercism. The tests below might not correspond to the solution code, and the exercise may have changed.
```

Comment:
Can I meet you half way? :)

```suggestion
                  This solution was written on an old version of Exercism. The tests below might not correspond to the solution code, and the exercise may have changed since this code was written.
```